### PR TITLE
{math}[foss/2021a] retworkx v0.9.0 w/ Python 3.9.5

### PR DESCRIPTION
**Pull request by Micket**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
alvis-c1 - Linux centos linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake), Python 3.6.8
See https://gist.github.com/f5af21cf3e8921d6084736b17a9ae010 for a full test report.

File: easybuild/easyconfigs/r/retworkx/retworkx-0.9.0-foss-2021a.eb [hunk 1]
```diff
@@ -0,0 +1,31 @@
+easyblock = "PythonPackage"
+
+name = 'retworkx'
+version = '0.9.0'
+
+homepage = 'https://github.com/Qiskit/retworkx'
+description= """retworkx is a general purpose graph library for python3 written in Rust to take advantage of the
```
Comment (verdurin) [inline easybuild/easyconfigs/r/retworkx/retworkx-0.9.0-foss-2021a.eb:7]: Please fix the formatting here.

File: easybuild/easyconfigs/r/retworkx/retworkx-0.9.0-foss-2021a.eb [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+performance and safety that Rust provides. It was built as a replacement for qiskit's previous (and current)
+networkx usage (hence the name) but is designed to provide a high performance general purpose graph library for any
+python application. The project was originally started to build a faster directed graph to use as the underlying data
+structure for the DAG at the center of qiskit-terra's transpiler, but it has since grown to cover all the graph usage
+in Qiskit and other applications."""
+
+toolchain = {'name': 'foss', 'version': '2021a'}
+
+sources = [SOURCELOWER_TAR_GZ]
```
Comment (verdurin) [inline easybuild/easyconfigs/r/retworkx/retworkx-0.9.0-foss-2021a.eb:16]: Please add the missing checksum here.

**Review by verdurin — CHANGES_REQUESTED** [review]

**Comment by Micket** [comment]
@boegelbot please test @ generoso

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
easybuild-c7.novalocal - Linux centos linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/c70153f733d07c905550a9c8e9c101e3 for a full test report.

**Review by verdurin — APPROVED** [review]
Looks fine.

**Comment by verdurin** [comment]
Going in, thanks @Micket!

**Pull request merged**